cfme/containers/provider/openshift.py

```diff
@@ -40,7 +40,7 @@ def configloader(prov_config, prov_key):
         token_creds = OpenshiftProvider.process_credential_yaml_key(
             prov_config['credentials'], cred_type='token')
         return OpenshiftProvider(
-            name=prov_config['name'],
+            name=prov_config['name'].encode('utf-8'),
```

Comment:
Yeah, I was lazy there. We should convert all the loaded values to the utf-8. yaml has a way to do it

    import yaml
    def custom_str_constructor(loader, node):
        return loader.construct_scalar(node).encode('utf-8')
    yaml.add_constructor(u'tag:yaml.org,2002:str', custom_str_constructor)

but I didn't make it work together with the `yaycl`.

Maybe there is a way to walk trough the config tree after loading and convert all to utf, but I haven't found a way to do that yet.